network/stellar-disbursement-platform/admin-guide/48-tenant-provisioning.mdx

```diff
@@ -0,0 +1,98 @@
+---
+id: tenant-provisioning
+title: Provisioning a New Tenant
+description: Learn how to provision and configure a new tenant in the Stellar Disbursement Platform.
+keywords: [SDP, configuration, tenant, provisioning, single-tenant, multi0tenant]
+sidebar_position: 48
+---
+
+This section covers the steps to provision a new tenant in the Stellar Disbursement Platform (SDP). A tenant is an isolated organization or entity that uses the SDP to manage disbursements. Each tenant has its own configuration settings, users, disbursements, and mostly importantly, its own source of funds.
+
+The endpoints needed to manage tenants can be found in the [Admin (Tenant Management)](../api-reference/resources/admin) docs.
+
+## Creating Tenants
+
+Tenants can be created through the [POST /tenants](../api-reference/resources/create-tenant) endpoint. As a quickstart, you can execute the [main.sh](https://github.com/stellar/stellar-disbursement-platform-backend/blob/develop/dev/main.sh) script from our repository and make sure to replace it with the desired values.
+
+The following sections detail how each field is used on the provisioning process and, so you can ensure the tenants are provisioned as expected:
+
+### `distribution_account_type`
+
+This is by far the most important field, as it determines the source of funds (distribution account) for the tenant, as well as how the secret for this distribution account is stored. The possible values are described below:
```

Comment:
Maybe it's worth mentioning here that this configuration cannot be changed once a tenant is created.